packages/docs/src/examples/WalletConnectExample.tsx

```diff
@@ -36,7 +36,7 @@ function App() {
           infuraId: '62687d1a985d4508b2b7a24827551934',
         })
         await provider.enable()
-        activate(provider)
+        await activate(provider)
```

Comment:
I think that it does not make any difference, but if linter shouts, we can add that.

Reply:
Yep, that is the point of `no-floating-promises` rule - you have to either await promise or use the `void` operator to explicitly say that you don't want to await the promise. I think here it makes more sense to await the promise.